.storybook/manager-head.html

```diff
@@ -38,6 +38,11 @@
   .os-content button:hover svg {
     color: #5034ff;
   }
+
+  /* https://github.com/storybookjs/storybook/issues/25094 */
+  div[data-radix-scroll-area-viewport] > div {
+    display: block !important;
```

Comment:
Unfortunately it's inline style and no selectors, so it's 🤮 
Hoping it's temp until the bug is fixed on their side

Reply:
This is the best we can do